### ISSUE TITLE
Internationalization

### ISSUE THREAD

**Issue by whitequark** [solvespace/solvespace#112]
This requires:
  * writing a gettext runtime implementation that's not as gross as gettext itself (and works properly on win32 anyhow, and doesn't need po→mo translation);
  * marking menus and the like with `_(...)` (easy);
  * localizing the Printf-based text window (hard).

**Comment by Symbian9** [comment]
Maybe simple textfiles, similar to how its realised in PolygonCrest and YSFlight?
* https://github.com/YSFlight-opensource/PolygonCrest
* https://github.com/YSFlight-opensource/ysflight-languages

**Comment by whitequark** [comment]
No, the gettext format will be used. The advantage is that there is a lot of existing tools, like xgettext and poedit, that understand them.

**Comment by whitequark** [comment]
@Symbian9 I've applied your ua-UA translation. I cannot reply to your email because your mail host rejects my mail with:
```
<appsoft@ua.fm>: host mx6.ua.fm[91.198.36.62] said: 550 Host in a black list.
    See http://mail.i.ua/err/9/?ip=188.166.218.19 (in reply to RCPT TO command)
Reporting-MTA: dns; uruz.whitequark.org
X-Postfix-Queue-ID: 2EFC9203A9
X-Postfix-Sender: rfc822; whitequark@whitequark.org
Arrival-Date: Fri,  6 Jan 2017 22:29:51 -0500 (EST)

Final-Recipient: rfc822; appsoft@ua.fm
Action: failed
Status: 5.0.0
Remote-MTA: dns; mx6.ua.fm
Diagnostic-Code: smtp; 550 Host in a black list. See
    http://mail.i.ua/err/9/?ip=188.166.218.19
```

**Comment by Symbian9** [comment]
@whitequark, thanks! 

Think, problem is with your own mail server, that not recognised by my mail hosting.
Try use <code>Symbian9@github.com</code>
* https://developer.github.com/v3/users/emails/

**Comment by whitequark** [comment]
> Think, problem is with your own mail server, that not recognised by my mail hosting.

No. Every other mail hoster so far had no problems with my server. Complain to your hoster.

> It should be uk_UA!!!

Fixed

**Comment by Symbian9** [comment]
> No. Every other mail hoster so far had no problems with my server. Complain to your hoster.

@whitequark, When somebody send me message from GMail I has on problems with getting them. Also, Github & Reddit, governmental mail servers normally send mails to described above my mail host. So, its only issue in your mail hoster, that prevent sent messages from you to me.

**Comment by whitequark** [comment]
> So, its only issue in your mail hoster, that prevent sent messages from you to me.

No. There is no reason for ua.fm to reject my mail, and I don't care what the excuses are.
